### PR TITLE
Default genesis parameters yielding positive treasury

### PR DESCRIPTION
**Pull request by carbolymer**
# Changelog

```yaml
- description: |
    Default genesis parameters yielding positive treasury
# uncomment types applicable to the change:
  type:
  # - feature        # introduces a new feature
  # - breaking       # the API has changed in a breaking way
   - compatible     # the API has changed but is non-breaking
  # - optimisation   # measurable performance improvements
  # - improvement    # QoL changes e.g. refactoring
  # - bugfix         # fixes a defect
  # - test           # fixes/modifies tests
  # - maintenance    # not directly related to the code
  # - release        # related to a new release preparation
  # - documentation  # change in code docs, haddocks...
```

# Context

Without non-zero rho and tau, treasury is not growing.

# Checklist

- [ ] Commit sequence broadly makes sense and commits have useful messages
- [ ] New tests are added if needed and existing tests are updated. See [Running tests](https://github.com/input-output-hk/cardano-node-wiki/wiki/Running-tests) for more details
- [ ] Self-reviewed the diff

### CODE REVIEW

File: cardano-api/internal/Cardano/Api/Genesis.hs [hunk 1]
```diff
@@ -129,6 +134,8 @@ shelleyGenesisDefaults =
     , sgMaxLovelaceSupply     = 0
     }
   where
+    unsafeUI :: Rational -> Ledger.UnitInterval
```
Comment (Jimbo4350) [inline cardano-api/internal/Cardano/Api/Genesis.hs:137]: Ledger has: [unsafeBoundRational](https://github.com/IntersectMBO/cardano-ledger/blob/92fd6688b3cce015abc46c133c68e6ea7baa6503/libs/cardano-ledger-core/testlib/Test/Cardano/Ledger/Core/Rational.hs#L21)

Comment (carbolymer) [reply cardano-api/internal/Cardano/Api/Genesis.hs:137]: But that's a test module. Do we want to introduce a dependency on a test module?

File: cardano-api/internal/Cardano/Api/Ledger/Types.hs [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+-- | Useful utilities for handling ledger types
+module Cardano.Api.Ledger.Types where
+
+import           Cardano.Ledger.BaseTypes
+
+import           Data.Maybe
+import           Data.Typeable
+import           GHC.Stack
+
+-- | Convert Rational to a bounded rational. Throw an exception when the rational is out of bounds.
```
Comment (Jimbo4350) [inline cardano-api/internal/Cardano/Api/Ledger/Types.hs:14]: we have `Cardano.Api.Utils`

**Review by Jimbo4350 — APPROVED** [review]
LGTM but move the new function to `Cardano.Api.Utils`

File: cardano-api/internal/Cardano/Api/Ledger/Types.hs (hunk 2 above)
Comment (carbolymer) [reply cardano-api/internal/Cardano/Api/Ledger/Types.hs]: moved there

**Pull request merged**